qiskit/algorithms/observables_evaluator.py

```diff
@@ -0,0 +1,157 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Evaluator of auxiliary operators for algorithms."""
+from __future__ import annotations
+
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.opflow import PauliSumOp
+from . import AlgorithmError
+from .list_or_dict import ListOrDict
+from ..primitives import EstimatorResult, BaseEstimator
+from ..quantum_info.operators.base_operator import BaseOperator
+
+
+def estimate_observables(
+    estimator: BaseEstimator,
+    quantum_state: QuantumCircuit,
+    observables: ListOrDict[BaseOperator | PauliSumOp],
+    threshold: float = 1e-12,
+) -> ListOrDict[tuple[complex, tuple[complex | None, int | None]]]:
```

Comment:
From what I can tell, in the absence of Estimator metadata entirely, or specific aspects variance and/or shot, these are set with defaults of 0,0 so None does not seem to ever be in this tuple. Removing None would save the user checking as it seems this will either be some valid valid (which might be 0 for variance) or 0.